src/pages/api/oauth/accept-consent.ts

```diff
@@ -2,7 +2,7 @@ import { gql } from 'graphql-request'
 import { NextApiRequest, NextApiResponse } from 'next'
 
 import { createGraphqlFetch } from '@/api/graphql-fetch'
-import { AuthSessionCookie } from '@/auth/auth-session-cookie'
+import { kratos } from '@/auth/kratos'
 
 export const config = {
```

Comment:
Aside your code, this experimental edge (that I may have added after testing), it causing at least in localhost that query in req.query (line 15) is undefined.

Reply:
I did not touch the `experimental-edge` setting or `req.query`.
Not sure what is different then. Sure the same test in `kratos` branch has a different outcome?

Reply:
> Sure the same test in kratos branch has a different outcome?

Sorry, I meant that also in `kratos` we have this issue (BTW only with localhost frontend and local dev env) and we could remove it.

Reply:
ahhh, now it makes sense :)
The syntax is a bit different for the edge routes (see [ea50fb1](https://github.com/serlo/frontend/pull/1927/commits/ea50fb1c439880e2f3c55525483fac3da2cd2fce))

Reply:
>  (BTW only with localhost frontend and local dev env)

I have to correct myself here: localhost frontend, also against staging backend services.

Reply:
@elbotho I'll remove them anyway, I'm getting unsure about it after so many warnings that the edge API may change

Reply:
Alright, if you do it in this branch you'd have to revert my updates.

Reply:
> It would also be hard to debug, if they sunddelly change it...

I follow nextjs changelogs quite closely and they usually communicate those things quite well and the types would also change. So either way I'm not that worried.